config/honeybadger.yml

```diff
@@ -1,2 +1,2 @@
 ---
-api_key: <%= Settings.honeybadger_api_key %>
+api_key: <%= ENV['HONEYBADGER_API_KEY'] %>
```

Comment:
This needs to be reverted. I do not want to look into initializer to see the values. That is why settings.yml was there. And same needs to be there in secretys.yml too.